Feature/custom style rename

firmusoft opened the pull request:
1. Fix: First word/sentence bold style is not working correctly
 2. Fix: Indent is having issues when applying along with level
 3. Rename Functionality for custom style
 4. Fix: On deselect bold,italics and underline is not reflected in example piece of style menu
 5. Custom style JSON property name changed

Seybi-ea committed:
fixed couple of  flow errors.

Seybi-ea committed:
1. Fix: Apply the None custom style not removing the applied custom styles
2. Fix: Numbering,indenting,paragraph spacing not reflecting in
on Sample text in Custom style editor.

Seybi-ea committed:
1. FIX: When apply 'None' from style menu, not clearing the applied custom style.
2. Numbering, indenting, paragraph spacing not reflecting on Sample text in Custom Editor UI.

Seybi-ea committed:
Modified the README.md file.

firmusoft committed:
Added new css file to handle paragraph numbering.

firmusoft committed:
Implemented paragraph numbering

firmusoft committed:
Removed text color from number content

firmusoft committed:
Added style to number content.

firmusoft committed:
Fix:Copy/paste custom style numbering.

firmusoft committed:
Handled to apply edited custom style wherever it is used in editor .

Seybi-ea committed:
1. Issue fix : Duplicate style created while modified the style name.

Seybi-ea committed:
Handle enter key (Apply the same style to the new paragraph).

Seybi-ea committed:
Bring custom style for a blank line(commit without completion)

firmusoft committed:
Handled new line style with markson enter key press

Seybi-ea committed:
1. Fix: Underline not working as expected , If user unselect underline
from a custom style it didn't reflect to the editor.
2. Fix: Showing multiple submenus at a time.
3. Fix: Issue fix: sub popup not closing after selecting another(shows
multiple pop ups at a time)

Seybi-ea committed:
1. Fix: Losing focus from the editor when applying a custom Style
2. Fix: Unable to apply a style in a blank line.

Seybi-ea committed:
Handled clear style in empty line

Seybi-ea committed:
Clear marks for empty line on None click

Seybi-ea committed:
Merge conflict

firmusoft committed:
Fix:Capco removes from node on enter key press.

arathygopal-gs committed:
1. Sort stylenames alphabetically
2. Custom style editor UI changes

Seybi-ea committed:
1.Fix:Bullet list not working as expected on indent
2.Fix:Styling issues in style editor

arathygopal-gs committed:
Fix: Selected Text align not showing correctly.

Seybi-ea committed:
1.Added tool tip for style select box

Seybi-ea committed:
Removed apply style from style plugin on document load

Seybi-ea committed:
1.Bold for number part in custom style numbering
2.fixed transaction mismatch error on loading snapshot
3.fixed alphanumeric list issue with bracket.

Seybi-ea committed:
Added functionality for inserting empty elements for custom style numbering when hierachy breaks

firmusoft committed:
Fixed issues in creating empty element for custom style numbering

Seybi-ea committed:
Added blank element below the current selection when heirarchy breaks for numbered style paragraphs.

Seybi-ea committed:
Added indent behaviour for numbered custom style

Seybi-ea committed:
Added option for applying bold for sentence and first word.

arathygopal-gs committed:
1. Server created to handle custom styles globally.
2. Sonar issue fixes.
3. README.md file modifications to include the commands to start custom style server.
4. Clinet side changes related to the new server implementation to save/retireve the custom styles.

arathygopal-gs committed:
Placed a list of existing custom styles in CustomStyle Editor
and populate the styles on clicking on a style in the list

arathygopal-gs committed:
1. FIX: saved custom styles shows in menu only after reloading editor
2. sonar issue fixes
3. FIX: select two paragraph with diffrent style applied,shows the last
applied style insted of None.
4.

Seybi-ea committed:
Issue fix on heirachy when applying none to a numbered custom style

arathygopal-gs committed:
Flow error fixes.

arathygopal-gs committed:
1. Made corrections with respect to the reviewer's comments:
   1. Don't leave commented out code in the project unless you are intending to restore it
   2. Avoid unnecessary if..else statements when returning boolean

arathygopal-gs committed:
Missed to check one condition in Styleplugin.

arathygopal-gs committed:
1. Rename implementation
2. FIX: Indent overriding not working on a paragraph where cussom style is applied
3. sonar issue fix
4. Handled numbering heirarchy on delete

Commit:
1. Loader for image assets (especially from plugins).
2. Needn't call DocAttrStep.Register from Collab Server as it is self invoked.

arathygopal-gs committed:
1. Disabled the style name textbox for modify custom style.
2. FIX: Able to save a new custom style with already exist style name .
3. FIX: Should not load style in Create Style window on initial load.

melgish committed:
changes for demo style server

melgish committed:
rename docker file and add notes

melgish committed:
fault tolerant style server

melgish committed:
graceful shutdown on SIGTERM or SIGINT

melgish committed:
build style-service without python

melgish committed:
better comments

melgish committed:
issues reading / writing in docker

arathygopal-gs committed:
1. FIX: Not able to modify and save the populated style in Custom Style Editor.

firmusoft committed:
Merge pull request #50 from MO-Movia/feature/demo-docker

Feature/demo docker

Seybi-ea committed:
1. Load document with existing style name.
2. Fix- undefined object from custom style server.

Seybi-ea committed:
1. fix:Return type inconsistent in readStyles function of custom style server

melgish committed:
learn how reduce works!

melgish committed:
add simple import / export

Seybi-ea committed:
1. Fix: First word/setence bold style is not working correctly
2. Fix: Indent is having issues when applying along with level
3. Sonar issue fix on css
4. Fix: On deselect bold,italics and underline is not reflected in example piece of style menu
5. Custom style JSON property name changed

melgish reviewed (commented):
Any merge issues between main and this branch need to be resolved.

firmusoft committed:
Merge branch 'main' into feature/custom_style_rename

Seybi-ea committed:
1.Style server readStyles method changed to read both new and old JSON format.
2.Fix: Bold first word/sentence not clearing on None/Clear command.
3.Soanr fix droped Bugs count to 5.
